views/extended-public-key.pug

```diff
@@ -24,11 +24,10 @@ block content
 			+summaryItem("BIP32 Path", "BIP32 defines a standard for deriving child keys from an extended parent key. This 'path', also known as the 'derivation path', comes from that standard and instructs software how to generate child keys.")
 				| #{bip32Path}
 
-			if (balance)
-				+summaryItem("Balance")
-
-			if (changeBalance)
-				+summaryItem("Change Balance")
+			+summaryItem("Balance")
+				span(class=(balanceSat > 0 ? "text-success" : false))
```

Comment:
The challenge with the balance appearing here is that it's the balance of the first 20 receive and 20 change addresses. A business wallet (or power user) could have more addresses. Somehow the user would need to know the limitations of this balance calculation.

Reply:
Note that @janoside has subsequently clarified balance meaning by displaying current page address range  (cf. commit 9fdc075)